modules/ui/fields/roadspeed.js

```diff
@@ -113,16 +119,20 @@ export function uiFieldRoadspeed(field, context) {
     roadspeed.tags = function(tags) {
         _tags = tags;
 
-        var value = tags[field.key];
+        var rawValue = tags[field.key];
+        var value = rawValue;
         var isMixed = Array.isArray(value);
 
         if (!isMixed) {
-            if (value && value.indexOf('mph') >= 0) {
-                value = parseInt(value, 10).toString();
-                _isImperial = true;
-            } else if (value) {
+            if (rawValue && rawValue.indexOf('mph') >= 0) {
+                _isImperial = rawValue && rawValue.indexOf('mph') >= 0;
```

Comment:
can't this be simplified to 
```suggestion
                _isImperial = true;
```
?